### PR TITLE
Export keras merge layer.

### PR DESCRIPTION
**Pull request by dargor**
This layer is accessible in raw Keras API, and useful to build custom merge layers.

Also fixed a typo preventing wheel from building successfully.

### CODE REVIEW

**Comment by dargor** [comment]
Rebasing since `setup.py` is fixed on master.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @case540: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @case540: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @case540: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by mihaimaruseac** [comment]
@case540 no longer works on tensorflow, so taking over.

@martinwicke @karmel @fchollet Does it make to export that symbol?

**Comment by karmel** [comment]
No-- @dargor , can you provide an example of what you're trying to do?

**Comment by dargor** [comment]
The final layer of a DDQN, for example : 

```python
class Q(_Merge):

    def _merge_function(self, inputs):
        advantage, value = inputs
        return value + advantage - K.mean(advantage, axis=-1, keepdims=True)
```

**Comment by fchollet** [comment]
Thank you for the PR. This is a private API, we will not export it.

Your use case should be doable with a custom layer as such:

```python
class YourMerge(Layer):

    def call(self, inputs):
        advantage, value = inputs
        return value + advantage - K.mean(advantage, axis=-1, keepdims=True)
```
